docker-app/qfieldcloud/core/drf_utils.py

```diff
@@ -0,0 +1,104 @@
+from django.db.models import QuerySet
+from django.http import HttpRequest
+from rest_framework import filters, viewsets
+
+
+class QfcOrderingFilter(filters.OrderingFilter):
+    """
+    Custom QFC OrderingFilter class that allows to use custom expression
+    Use it in a ModelViewSet by setting the `filter_backends` and `ordering_fields` fields
+    It is possible to use an `ordering_fields` expression value with attributes
+    e.g.: "my_field::alias=my_field_alias,key=value"
+    """
+
+    SEPARATOR = "::"
+    TOKENS_LIST_SEPARATOR = ","
+    TOKENS_VALUE_SEPARATOR = "="
+
+    def _get_query_field(self, fields: list[str], term: str) -> str | None:
+        """
```

Comment:
Generic comment for the documentation:
- please add a dot at the end of the sentences :)
- Make sure there the first line is a concise summary of what the function is doing, add a new empty line and continue with a more detailed explanation.
- Follow with arguments and return value documentation too.

See some docs examples from here: https://github.com/opengisch/QFieldCloud/blob/master/docker-app/qfieldcloud/core/utils2/storage.py

Reply:
Sure, let's follow the convention